scalardb-analytics-postgresql-sample/sample-data-loader/src/main/java/sample/data/Loader.java

```diff
@@ -0,0 +1,183 @@
+package sample.data;
+
+import com.scalar.db.api.DistributedTransaction;
+import com.scalar.db.api.DistributedTransactionManager;
+import com.scalar.db.api.Put;
+import com.scalar.db.exception.transaction.TransactionException;
+import com.scalar.db.io.Key;
+import com.scalar.db.service.TransactionFactory;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.function.Function;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVRecord;
+
+public class Loader implements AutoCloseable {
+
+  private static final String[] CUSTOMER_COLUMNS = {
+    "c_custkey",
+    "c_name",
+    "c_address",
+    "c_nationkey",
+    "c_phone",
+    "c_acctbal",
+    "c_mktsegment",
+    "c_comment"
+  };
+
+  private static final String[] ORDERS_COLUMNS = {
+    "o_orderkey",
+    "o_custkey",
+    "o_orderstatus",
+    "o_totalprice",
+    "o_orderdate",
+    "o_orderpriority",
+    "o_clerk",
+    "o_shippriority",
+    "o_comment"
+  };
+
+  private static final String[] LINEITEM_COLUMNS = {
+    "l_orderkey",
+    "l_partkey",
+    "l_suppkey",
+    "l_linenumber",
+    "l_quantity",
+    "l_extendedprice",
+    "l_discount",
+    "l_tax",
+    "l_returnflag",
+    "l_linestatus",
+    "l_shipdate",
+    "l_commitdate",
+    "l_receiptdate",
+    "l_shipinstruct",
+    "l_shipmode",
+    "l_comment"
+  };
+
+
+
```

Comment:
Thank you for pointing this out. I normally leave this kind of styling to formatters. But, since this is a small project, I skipped adding it 😢 . In https://github.com/scalar-labs/scalardb-samples/pull/56/commits/5032262ed8f785a80d4d2a3d8c6c1460230529a3, I added the spotless to the project, and apply it to all codes.